components/ai_chat/resources/page/state/data-context-provider.tsx

```diff
@@ -19,8 +19,91 @@ function toBlobURL(data: number[] | null) {
 const MAX_INPUT_CHAR = 2000
 const CHAR_LIMIT_THRESHOLD = MAX_INPUT_CHAR * 0.80
 
+const ACTIONS_LIST: ActionsList[] = [
```

Comment:
As discussed, let's put this in C++ (or via mojom)

Reply:
sending list from c++ now